app/constants/settings.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {t} from '@i18n';
+
+export const SettingOptionConfig = {
+    notification: {
+        defaultMessage: 'Notifications',
+        i18nId: t('user.settings.modal.notifications'),
+        iconName: 'bell-outline',
+        testID: 'general_settings.notifications.action',
+    },
+    display: {
+        defaultMessage: 'Display',
+        i18nId: t('user.settings.modal.display'),
+        iconName: 'layers-outline',
+        testID: 'general_settings.display.action',
+    },
+    advanced_settings: {
+        defaultMessage: 'Advanced Settings',
+        i18nId: t('mobile.advanced_settings.title'),
+        iconName: 'tune',
+        testID: 'general_settings.advanced.action',
+    },
+    about: {
+        defaultMessage: 'About {appTitle}',
+        i18nId: t('about.title'),
+        iconName: 'information-outline',
+        testID: 'general_settings.about.action',
+    },
+    help: {
+        defaultMessage: 'Help',
+        i18nId: t('mobile.help.title'),
+        testID: 'general_settings.help.action',
+        showArrow: false,
+    },
```

Comment:
you have the opportunity to name the id's and test Ids in a way that makes sense with your code, I know you took this from v1 and that is fine and it will probably alleviate some translators and maybe e2e tests, but I don't think at this moment this `ids` make much sense.

up to you, 0/5 on my part.

Reply:
updated